backend/src/hatchling/build.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import annotations
+
 import os
+from typing import Any
 
 
-def get_requires_for_build_sdist(config_settings=None):
+def get_requires_for_build_sdist(config_settings: dict[str, Any] | None = None) -> list[dict[str, Any]]:
```

Comment:
returns list of strings https://peps.python.org/pep-0517/#get-requires-for-build-sdist

Reply:
fixed.